src/server.py

```diff
@@ -148,6 +156,38 @@ def binify(bins, data):
     return result
 
 
+def valve(head_rev, pushlog_id, branch, priority):
+    """Determine which kind of job should been returned"""
+    priority = priority
+    BRANCH_COUNTER.increase_the_counter(branch)
```

Comment:
we should make sure that 'branch' is valid- we will get called for EVERY decision task.  One way to reduce this is to make sure that we hardcode the list of SETA branches in the in-tree code for the decision task where we call SETA server.  But still, this needs to be 100% robust.